beetmoverscript/templates/firefox_candidates_repacks.yml

```diff
@@ -47,6 +47,14 @@ mapping:
       s3_key: {{ platform }}/{{ locale }}/firefox-{{ version }}.tar.bz2.asc
       destinations:
         - {{ platform }}/{{ locale }}/firefox-{{ version }}.tar.bz2.asc
+    SOURCE:
```

Comment:
I don't think this is necessary. This is template for firefox_candidates_repacks which is not the case here. I suppose an en-US beetmover linux/linux64 based job will transfer this, hence this section is not needed.